cypress/e2e/read-receipts/high-level.spec.ts

```diff
@@ -168,7 +168,23 @@ describe("Read receipts", () => {
             // The room is unread again
             assertUnread(room2, 1);
         });
-        it.skip("A receipt for the last unimportant event makes the room read, even if all are unimportant", () => {});
+        it("A receipt for the last unimportant event makes the room read, even if all are unimportant", () => {
```

Comment:
I'm wondering if the intention of this test was that the room should start off *unread*?

@andybalaam as the guardian of these tests I think this is one for you

Reply:
No, this is just to say that unimportant events that are read don't make the room unread, so I think this is good modulo the comment I made.